x/posts/client/cli/tx.go

```diff
@@ -47,19 +49,41 @@ func GetTxCmd(_ string, cdc *codec.Codec) *cobra.Command {
 // GetCmdCreatePost is the CLI command for creating a post
 func GetCmdCreatePost(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create [subspace] [message] [allows-comments] [[[uri],[mime-type]]...]",
+		Use:   "create [subspace] [message] [allows-comments]",
 		Short: "Create a new post",
 		Long: fmt.Sprintf(`
 				Create a new post, specifying the subspace, message and whether or not it will allow for comments.
-				Optional media attachments are also supported.
-				If you with to add one or more media attachment, you have to specify a uri and a mime type for each.
+				Optional media attachments and polls are also supported.
+				If you want to add one or more media attachment, you have to use the --medias flag 
+				specifying after that a string containing couples of (URI,mime-type) separated by a space one from the other.
 				Each attachment can be added only once, otherwise and error will occur.
-                You can do so by concatenating them together separated by a comma (,).
 				Usage examples:
 
-				- tx posts create "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e" "Hello world!" true
-				- tx posts create "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e" "A post with media" true "https://example.com,text/plain"
-				- tx posts create "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e" "A post with multiple medias" false "https://example.com/media1,text/plain" "https://example.com/media2,application/json"
+				- tx posts create "desmos" "Hello world!" true
+                - tx posts create "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e" "A post with media" false \
+				  --medias "https://example.com/media1,text/plain"
+				- tx posts create "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e" "A post with multiple medias" false \
+                  --medias "https://example.com/media1,text/plain https://example.com/media2,application/json"
```

Comment:
No problem for the queries, after our last conversations about poll's implementation I removed all the code related to them and forgot to delete them!

For the CLI command actually we can post medias in both ways:
mine and as you explained here:
``` 
>   tx posts create "..." "A post with multiple medias" false \
>   --media "https://example.com/media1,text/plain"
>   --media "https://example.com/media2,application/json" 
```
For the 2) We could do like I've done for the poll details, I think it could work!

Reply:
I think it would be better to delete the `--medias` flag and just leave the `--media` one. Even better would be change the `--media` implementation using the map inside it